bears/general/IndentationBear.py

```diff
@@ -118,12 +129,16 @@ def run(self,
         if new_file != list(file):
             wholediff = Diff.from_string_arrays(file, new_file)
             for diff in wholediff.split_diff():
-                yield Result(
-                    self,
-                    'The indentation could be changed to improve readability.',
-                    severity=RESULT_SEVERITY.INFO,
-                    affected_code=(diff.range(filename),),
-                    diffs={filename: diff})
+                if not any(gt_eq(diff.range(filename).start, string.range.start)
```

Comment:
switch the logic to be `if any(...): continue` , then you do not need to touch the existing `yield` lines.